packages/legacy-client/src/client/objectSets/createObjectSetSearchAround.ts

```diff
@@ -25,6 +25,7 @@ import type { SearchAround } from "../interfaces/searchAround";
 import type { OsdkLegacyObjectFrom } from "../OsdkLegacyObject";
 import { createOsdkObjectSet } from "./OsdkObjectSet";
 
+/**@deprecated */
```

Comment:
How do I actually deprecate the searchAround functions we generate on line 41?

Reply:
Like `searchAroundCar`, `searchAroundPlane` etc.

Reply:
Gotta change the code gen!